webpack/react_app/components/FeaturesDropdown/index.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  DropdownItem,
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleAction,
+} from '@patternfly/react-core';
+import { push } from 'connected-react-router';
+
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { STATUS } from 'foremanReact/constants';
+
+import { REX_FEATURES_API, NEW_JOB_PAGE } from './constant';
+import { runFeature } from './actions';
+
+const FeaturesDropdown = ({ hostId }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const {
+    response: { results: features },
+    status,
+  } = useAPI('get', foremanUrl(REX_FEATURES_API));
+  const dispatch = useDispatch();
+  const dropdownItems = features
+    ?.filter(feature => feature.host_action_button)
```

Comment:
@adamruzicka I tried to filter this by the API call, with no success
I added `scoped_search :on => :host_action_button` 
and created an API call to `/api/remote_execution_features?search=host_action_button=true` 

We can filter it on the client, but if the server can return just the relevant features, it just better :)

Reply:
If I'm reading things right the controller currently does not support searching, tracking it as[1]. We could implement that, but for now let's go with frontend only filtering. Standard Satellite deployment has around 20 features, so nothing terrible.

[1] - https://projects.theforeman.org/issues/34284